project-base/storefront/pages/order/contact-information.tsx

```diff
@@ -32,7 +33,16 @@ export const getServerSideProps = getServerSidePropsWrapper(
                 redisClient,
                 domainConfig,
                 t,
-                prefetchedQueries: [{ query: CountriesQueryDocument }],
+                prefetchedQueries: [
+                    { query: CountriesQueryDocument },
+                    {
+                        query: AdvertsQueryDocument,
+                        variables: {
+                            positionName: 'cartPreview',
```

Comment:
why is necessary to prefetch adverts with position cartPreview on the contact-information page?

Reply:
The need to prefetch this comes from `OrderSummary` component, which uses `<Adverts ... positionName="cartPreview" />`. This cascades down on these children components `ContactInformationPage -> ContactInformationWrapper -> OrderContentWrapper -> OrderSummary -> Adverts`. If I remove it I can see it is being fetched on the FE (in the networks tab).